src/styles/components/_team-page.scss

```diff
@@ -10,32 +10,48 @@
       margin: 0.75rem 0;
     }
   }
+  &__team {
+    margin-bottom: 5px;
+  }
   &__team__members {
     display: flex;
     flex-wrap: wrap;
     flex-direction: row;
     justify-content: center;
     &__member {
-      width: 200px;
-      margin: 1rem;
+      width: 220px;
+      margin: 0.4rem;
       text-align: center;
+      background-color: white;
+      border-radius: 2px;
+      //border: 1px solid rgba(0,0,0,0.2);
+      box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
+      transition: box-shadow 0.2s cubic-bezier(.25,.8,.25,1);
+      &:hover {
+        //border-color: $link-blue;
+        box-shadow: 0 8px 12px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+      }
       &__image {
         @include uppercase;
         object-fit: cover;
-        padding: 0 1rem 0 0;
-        width: 200px;
-        height: 200px;
-      }
-      &__name {
-        font-size: 1.3rem;
-        font-weight: 400;
-        margin: 0 0.75rem 0 0;
+        width: 220px;
+        height: 220px;
       }
-      &__job-title {
-        font-size: 1rem;
-        font-weight: 300;
-        color: $light-grey;
-        margin: 0 0.5rem 0 0;
+      &__text{
+        padding: 0.4rem;
+
+        &__name {
+          font-size: 1.3rem;
+          font-weight: 400;
+          color: black;
+          margin: 0;
+        }
+        &__job-title {
+          font-size: 0.8rem;
+          font-weight: 300;
+          color: rgba(0, 0, 0, 0.4);
```

Comment:
Same thing with hex here. If possible use an already defined `SCSS` variable from `./src/styles/abstracts/_variables.scss` like `$light-grey`

Reply:
Btw @zanemountcastle stylelint could also lint all these things if you were interested :)